specification/compute/resource-manager/readme.md

```diff
@@ -218,7 +218,7 @@ input-file:
 - Microsoft.Compute/stable/2020-12-01/compute.json
 - Microsoft.Compute/stable/2020-12-01/runCommands.json
 - Microsoft.Compute/stable/2019-04-01/skus.json
-- Microsoft.Compute/stable/2020-09-30/disk.json
+- Microsoft.Compute/stable/2020-12-01/disk.json
```

Comment:
Please generate the .Net SDK code and create tests for the new functionality, record those tests, and include the recorded json files in the PR. 
Please link to the .Net SDK PR here. The tests must all pass before this Swagger PR can be merged.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/19092

Reply:
@meet1993shah , have you investigated the .Net SDK failures? Seems like the build is failing.

Reply:
resolved the errors, build is passing